rt/vm/longs/src/main/scala/scala/scalajs/js/JavaScriptException.scala

```diff
@@ -0,0 +1,26 @@
+/**
+ * Back 2 Browser Bytecode Translator
+ * Copyright (C) 2012-2017 Jaroslav Tulach <jaroslav.tulach@apidesign.org>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. Look for COPYING file in the top folder.
+ * If not, see http://opensource.org/licenses/GPL-2.0.
+ */
+package scala.scalajs.js
+
+/** Placeholder with simplified hashCode */
+final case class JavaScriptException(exception: scala.Any) extends RuntimeException {
+  override def fillInStackTrace(): Throwable = {
+    this
+  }
+  override def hashCode(): Int = 1973;
```

Comment:
This works for me and removes the generated [MurmurHash3](https://github.com/jtulach/bck2brwsr/pull/7/commits/e91be586c307f001cfbbf7e7a529e578760aa369) code, @sjrd.  Scala.js could do something similar (with different constant, of course), or by returning `getMessage().hashCode()` for example.